src/client/pages/profile/addimage.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useContext } from "react";
+import Router from "next/router";
+import axios from "axios";
+import gql from "graphql-tag";
+import { Image, CloudinaryContext, Transformation } from "cloudinary-react";
+import { Mutation } from "react-apollo";
+import Layout from "@/client/components/layouts/Layout";
+import { AuthContext } from "@/client/context";
+
+export const ADD_IMAGE = gql`
+  mutation addImage($email: String, $imageId: String) {
+    addImage(email: $email, imageId: $imageId) {
+      email
+      imageId
+    }
+  }
+`;
+
+const AddImage = props => {
+  const value = useContext(AuthContext);
+  const email = value.email;
+  let selectedFile;
+
+  const [imageId, setImageId] = useState("");
+
+  const fileSelectedHandler = event => {
+    selectedFile = event.target.files[0];
+  };
+
+  const fileUploadHandler = async event => {
+    event.preventDefault();
+
+    const formData = new FormData();
+    formData.append("file", selectedFile);
+    formData.append("upload_preset", "aqzs3lv0");
```

Comment:
It's better to assign the hardcoded value to some constant to show what the value represents.